docs/Design/Null Checking.md

```diff
@@ -0,0 +1,168 @@
+# Null Checking
+
+A "null" variable is a variable without a stored value. 
+Commonly, uninitialized variables have a null value.
+These commands check whether a passed variable has a null value. 
+
+## Commands
+
+### `is null`
+
+`is null : variable`
+
+Creates a test that will return whether variable is null.
+
+### `is not null`
+
+`is not null : variable`
+
+Creates a test that will return false if the variable is null.

```

Comment:
The `is null` description said "whether"; this says "false if". Better to standardize.